### ISSUE TITLE
"Note Stepping" being used as a way to suppress good quality Notes. Requires simple fix.

### ISSUE THREAD

**Issue by beyondarmonia** [twitter/communitynotes#145]
"Note Stepping" ( as it is called within the Community) is when someone puts up a bogus note as a way to suppress votes on another note that they disagree with, regardless of it's validity.

The problem arises from the fact that notes are arranged by time of submission, with the newest note showing at the top. And raters are most likely to see the note at the top, and less likely to scroll down and see those below it, especially when the top note is a bogus one.

So all someone has to do, if they see a pending note they don't like is to submit another note, and the time it takes for the original note to become visible will get much longer ( if not impossible ) since it will stop receiving as many ratings.

Fix:

Just change the way pending notes are sorted in the pending notes area. Instead of sorting by new, either sort by random or sort by top. Best is to sort by random at first and then sort by top once a certain threshold has been exceeded.

**Comment by Andrewcpu** [comment]
Agree. Random spread, or a more algorithmic choice on what's being displayed first. 

There's statistics to be looked re: how much of a problem this really is though. The amount of votes total may smooth out any noise introduced by stepping

**Comment by tudopatroi** [comment]
Strongly disagree. By ordering newest note first or random the system is vulnerable to gaming by bots.

The only correct way is to display the notes in chronological order with the OLDEST ONE ON TOP. This because most raters start with the first one they see, the top one.

It is absurd that  now one takes time to write a good note with links etc, only to see it buried at the bottom under 5-10 other fake notes which are just copying its content or links. So the impersonators are scoring points and improve their ratings, and are later able to do real damage. Not good.

In addition, when rating notes or reasons why a note is not needed, if the oldest is seen first it will be easy to spot and reject duplicates. This will reduce the incentives for 10 people to post 10 almost identical NNN replies, if the 9 most recent will be rated themselves as not needed.

In other words 
1-real people need to be discouraged to add redundant notes 
2-bots and scammers which post duplicate info to increase their impact will have their impact actually reduced
